torch/_inductor/codegen/common.py

```diff
@@ -450,6 +450,39 @@ def remainder(a, b):
     def load_seed(name, offset):
         return ops.load(name, sympy.Integer(offset))
 
+class DeviceOverrides():
+
+    @classmethod
+    def py_import_get_raw_stream_as(self, name):
+        raise NotImplementedError()
+
+    @classmethod
+    def py_set_device(self, device_idx):
+        raise NotImplementedError()
+
+    @classmethod
+    def py_synchronize(self):
+        raise NotImplementedError()
+
+    @classmethod
+    def py_DeviceGuard(self, device_idx):
+        raise NotImplementedError()
+
+    @classmethod
+    def cpp_defAOTStreamGuard(self, name, stream, device_idx):
+        raise NotImplementedError()
+
+    @classmethod
+    def cpp_defStreamGuard(self, name, stream):
```

Comment:
CPP is not a current priority. Let's remove it first.

Reply:
CUDA backend need this in EnterCudaDeviceContextManagerLine, [line](https://github.com/etaf/pytorch-inductor-xpu/blob/b8ce05456cd6e332746165e469ab1e9048c3daf5/torch/_inductor/codegen/wrapper.py#L198C34-L198C34) , do you mean we should not remove all cuda-bias code in EnterCudaDeviceContextManagerLine this time?

Reply:
> CPP is not a current priority. Let's remove it first.

CUDA backend need this in EnterCudaDeviceContextManagerLine, [line](https://github.com/etaf/pytorch-inductor-xpu/blob/b8ce05456cd6e332746165e469ab1e9048c3daf5/torch/_inductor/codegen/wrapper.py#L198C34-L198C34) , do you mean we should not remove all cuda-bias code in EnterCudaDeviceContextManagerLine this time?

Reply:
How about adding `namespace` as a parameter? Its value can be either `aten` or `torch` and the default is `torch`.